solutions/USACO jan16-build-gates.mdx

```diff
@@ -0,0 +1,92 @@
+---
+id: usaco-596
+source: USACO Silver 2016 January
+title: Build Gates
+author: Maggie Liu
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_gates_silver_jan16.html)
+
+<Warning>
+
+The official solution stores the farm in a $2005 \times 2005$ boolean array with the origin at $(1002, 1002)$, but this is not sufficient. Since each fence segment is doubled, the maximum distance in each direction could be $2000$ units, causing an input such as $1000$ $\texttt{N}$s to go out of bounds. To not go out of bounds, an array of size $4003 \times 4003$ should be used: $1$ unit for the origin position, $2000$ units in each direction from the origin and $1$ more unit in each direction to ensure that the farm includes area beyond the fence.
```

Comment:
use `\cdot` instead of `\times`

Reply:
But the `\times` here is equivalent to "by" in specifying the dimensions of the array, like `N \times M` [here](https://usaco.guide/silver/ff#implementation). Should it be changed to `\cdot`?

Reply:
i'm thinking times as well???

Reply:
oops you're right.